### PR TITLE
[CI:DOCS] Update troubleshooting.md

### PR DESCRIPTION
**Pull request by bwduncan**
- Consistent code block highlighting
- Minor spelling, formatting.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM!

Can you add a signoff? `git commit --amend -s` and then a force-push should do it.

**Comment by rhatdan** [comment]
Please sign your commits

git commit -a -s --amend
git push --force
LGTM

**Comment by bwduncan** [comment]
Oh woops, sorry. I actually used GitHub's editor to make this changes because it made it easy to preview the result, but I will check out the repo and amend the commit.

**Comment by rhatdan** [comment]
I wish there was an easy way to sign from github, but I am not aware of any.

**Comment by umohnani8** [comment]
@bwduncan you have some trailing whitespaces, that's why the validate test is failing.

**Comment by bwduncan** [comment]
Oh my. Well that's the last time I use the github editor! Force-pushed, should be passing soon. Thanks!

**Comment by TomSweeneyRedHat** [comment]
@bwduncan you're enjoying one of my least favorite games, chasing the ever-elusive trailing white space.  You might try:

`sed -i 's/[[:space:]]*$//' filename.ext `

File: troubleshooting.md [hunk 1]
```diff
@@ -1110,50 +1135,51 @@ ls: cannot open directory '/dir1': Permission denied
 
 We follow essentially the same solution as in the previous
 troubleshooting tip:
- "_Container creates a file that is not owned by the regular UID_"
+
+    Container creates a file that is not owned by the regular UID
+
```
Comment (rhatdan) [inline troubleshooting.md:1140]: White space leak is here

Comment (bwduncan) [reply troubleshooting.md]: it was a tab, that's why searching for "\ +$" in vim didn't find it. thanks!

**Pull request merged**